heartbeat/docs/securing-heartbeat.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[securing-heartbeat]]
+= Securing Heartbeat
+
+[partintro]
+
+--
+The following topics describe how to secure communication between Heartbeat
+and other products in the Elastic stack:
+
+* <<securing-communication-elasticsearch>>
+* <<configuring-ssl-logstash>>
+
+--
+
+[[securing-communication-elasticsearch]]
+== Securing Communication With Elasticsearch
+include::../../libbeat/docs/https.asciidoc[]
+
+//ASK STEFFEN IF LOGSTASH EVEN MAKES SENSE WITH THIS BEAT
```

Comment:
one might use logstash with any beat for different reasons. Like event-routing, some other output, hide elasticsearch, do geo-ip in logstash (to figure out in which data-center service X is currently running for example)?